apps/client/src/components/Lessons/ManageLessonDetails.svelte

```diff
@@ -44,11 +44,11 @@
     selectedLesson = event.target.selectedIndex - 1;
   };
 
-  const handleSubmit = async (lessonInput) => {
+  const handleSubmit = async (event) => {
```

Comment:
We might wanna start providing type definitions for such complex parameters. It'll help give everyone context as well as help you via intellisense in local development.

Reply:
That's a good idea!